framework/graphics/fps_info.cpp

```diff
@@ -118,64 +105,16 @@ void FpsInfo::EndFrame(uint64_t frame)
-                                                     static_cast<uint64_t>(measurement_end_time_));
-                uint64_t total_frames = measurement_end_frame_ - measurement_start_frame_;
-                double   fps          = static_cast<double>(total_frames) / diff_time;
-
-                nlohmann::json file_content = { { "frame_range",
-                                                  { { "start_frame", measurement_start_frame_ },
-                                                    { "end_frame", measurement_end_frame_ },
-                                                    { "frame_count", total_frames },
-                                                    { "start_time_monotonic", start_time },
-                                                    { "end_time_monotonic", end_time },
-                                                    { "duration", diff_time },
-                                                    { "fps", fps },
-                                                    { "frame_durations", frame_durations_ } } } };
-
-                FILE*   file_pointer = nullptr;
-                int32_t result       = util::platform::FileOpen(&file_pointer, measurement_file_name_.c_str(), "w");
-                if (result == 0)
-                {
-                    const std::string json_string = file_content.dump(util::kJsonIndentWidth);
-
-                    // It either writes a fully valid file, or it doesn't write anything !
-                    if (!util::platform::FileWrite(json_string.data(), json_string.size(), file_pointer))
-                    {
-                        GFXRECON_LOG_ERROR("Failed to write to measurements file '%s'.",
-                                           measurement_file_name_.c_str());
-
-                        // Try to delete the partial file from disk using <cstdio>
-                        const int remove_result = std::remove(measurement_file_name_.c_str());
-                        if (remove_result != 0)
-                        {
-                            GFXRECON_LOG_ERROR("Failed to remove measurements file '%s' (Error %i).",
-                                               measurement_file_name_.c_str(),
-                                               remove_result);
-                        }
-                    }
-                    util::platform::FileClose(file_pointer);
-                }
-                else
-                {
-                    GFXRECON_LOG_ERROR(
-                        "Failed to open measurements file '%s' (Error %i).", measurement_file_name_.c_str(), result);
-                    GFXRECON_LOG_ERROR("%s", std::strerror(result));
-                }
-            }
+            measurement_end_boot_time_    = util::datetime::GetBootTime();
+            measurement_end_process_time_ = util::datetime::GetProcessTime();
+            measurement_end_time_         = frame_end_time;
+            ended_measurement_            = true;
```

Comment:
Nit: When I ran I see this:

```
================== Start timer (Frame: 500) ==================
Load time:  0.000000 seconds (frame 1)
Total time: 106.769700 seconds
Measured FPS: 99.404368 fps, 5.029960 seconds, 500 frames, 1 loop, framerange [500-1000)
```

It would be nice if you added a "End timer" comment here as well because it just ends and I wasn't quite sure when the frame range covered while I was replaying.